miner_qpu.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 D-Wave Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from minorminer import find_embedding
+from dwave.system.samplers import DWaveSampler
+from dwave.system.composites import FixedEmbeddingComposite
+
+# Set up the QUBO. Start with the equations:
+# x + y - 2xy -1
+# 2yz - y - z
+# -2zx + z + x - 1
+# QUBO: 2x - 2xy + 2yz - 2zx - 2
+Q = {(0, 0): 2, (0, 1): -2, (0, 2): -2, (1, 2): 2}
+
+chainstrength = 2
+numruns = 1000
+
+dwave_sampler = DWaveSampler()
```

Comment:
Perhaps remove this line as you will be using the sampler later on?

Reply:
@m3ller, I need it to get the embedding. It's important in this example to show the embedding. Maybe I didn't understand your question?

Reply:
I saw your updated code. Please ignore my old suggestion.